public/live.php

```diff
@@ -266,22 +293,26 @@
 <?php include "../include/live/modals.php"; ?>
 <?php if ($my["id"] === $live["user_id"]) include "../include/live/add_blocking.php"; ?>
 <script id="com_tmpl" type="text/x-handlebars-template">
-  <div id="post_{{id}}" class="comment">
-    <div>
-      <img src="{{account.avatar}}" class="avatar rounded-circle" width="50" height="50" onclick="userDropdown(this, '{{id}}', '{{account.acct}}', '{{account.url}}')"/>
-    </div>
-    <div class="content">
-      <span onclick="userDropdown(this, '{{id}}', '{{account.acct}}', '{{account.url}}')" class="name">
-        {{#if donator_color}}
-        <span class="badge badge-pill" style="background:{{donator_color}}">
-        {{/if}}
-        <b>{{account.display_name}}</b>
-        {{#if donator_color}}
+  <div id="post_{{id}}" class="comment card mb-2">
+    <div class="content card-body">
+      <div class="float-left">
+        <img src="{{account.avatar}}" class="avatar rounded border" width="50" height="50" onclick="userDropdown(this, '{{id}}', '{{account.acct}}', '{{account.url}}')"/>
```

Comment:
```suggestion
        <img src="{{account.avatar}}" class="avatar rounded" width="50" height="50" onclick="userDropdown(this, '{{id}}', '{{account.acct}}', '{{account.url}}')"/>
```

Reply:
@Haritaso @yuzulabo I don't think this is necessary.

Reply:
👍